homeassistant/components/ambient_station/binary_sensor.py

```diff
@@ -65,6 +65,11 @@ def is_on(self):
 
         return self._state == 1
 
+    @property
+    def state(self):
```

Comment:
The binary sensor base component already implements `state` so we should only implement `is_on`.

Reply:
FYI, this was resolved in 4ea4a4d.

Reply:
Yeah, I was looking at a stale diff when I wrote that comment.